assets/queries/terraform/aws/security_group_rules_without_description/query.rego

```diff
@@ -0,0 +1,15 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_security_group[name]
```

Comment:
Shouldn't we check `object.get(resource, "description", "undefined") == "undefined"` too?